lib/internal/histogram.js

```diff
@@ -65,26 +68,26 @@ class Histogram {
     if (percentile <= 0 || percentile > 100)
       throw new ERR_INVALID_ARG_VALUE.RangeError('percentile', percentile);
 
-    return this.#handle ? this.#handle.percentile(percentile) : undefined;
+    return this[kHandle] ? this[kHandle].percentile(percentile) : undefined;
   }
 
   get percentiles() {
-    this.#map.clear();
-    if (this.#handle)
-      this.#handle.percentiles(this.#map);
-    return this.#map;
+    MapPrototypeClear(this[kMap]);
```

Comment:
Safe maps are safe to clear.
```suggestion
    this.#map.clear()
```